pkgs/top-level/all-packages.nix

```diff
@@ -23144,6 +23144,7 @@ with pkgs;
   openldap = callPackage ../development/libraries/openldap {
     openssl = openssl_legacy;
   };
+  openldap-libxcrypt-legacy = openldap.override { libxcrypt = libxcrypt-legacy; };
```

Comment:
Maybe it's worth overriding the description so things like s.n.o show the difference between the two?

Reply:
That's also not done for libxcrypt-legacy and the package is only here because otherwise it won't get cached.

This needs to go into 23.05 otherwise the portunus and the slapd it starts just crash loop without a proper indication what is wrong.